tardis/adapters/sites/moab.py

```diff
@@ -129,7 +137,6 @@ async def deploy_resource(
             remote_resource_uuid=remote_resource_uuid,
             created=datetime.now(),
             updated=datetime.now(),
-            drone_uuid=self.drone_uuid(remote_resource_uuid),
```

Comment:
@maxfischer2781 Sure. This line disables the change of the `drone_uuid`.

Reply:
Oh my, thanks! I completely missed that one! 🙈

Reply:
I have to admit, that It was hard to spot.